Kuroba/app/src/main/java/com/github/adamantcheese/chan/ui/captcha/v2/CaptchaNoJsHtmlParser.java

```diff
@@ -158,7 +158,7 @@ private void parseChallengeTitle(
             throw error;
         }
 
-        if (captchaTitle == null || captchaTitle.isEmpty()) {
```

Comment:
If captchaTitle is null then this change will throw a NullPointerException.

Reply:
The condition is always false, so I simplified it. I'll revert it to be safe.

Reply:
I've had weird shit come up in JSON more than enough times to not trust that a field is there even if I expect it; I'd rather be safe here.